graphene_linked_events/schema.py

```diff
@@ -414,10 +423,24 @@ class AddEventMutationInput(InputObjectType):
     )
 
 
-class UpdateEventMutationInput(AddEventMutationInput):
+class AddEventMutationInput(EventMutationInput):
+    draft = Boolean(
+        description="Set to `true` to save event as draft version, when draft is true, "
+        "event data validation will be skipped",
+        default_value=False,
+    )
+
+
+class UpdateEventMutationInput(EventMutationInput):
     id = String(required=True)
 
 
+class PublishEventMutationInput(UpdateEventMutationInput):
+    start_time = String(
+        description="This will be ignore since start_time will be the current time"
```

Comment:
Should be `ignored`. Also, I don't really understand this, if this parameter will be ignored, what is the point of having the parameter at all? 🙂

Reply:
yeah :D It's quite confusing here because the in `UpdateEventMutationInput` that field is required. Redeclare this field there so client doesn't have to include it in the mutation

Reply:
nvm, I think the other way around should be better to understand, fixed in the last commit